src/components/TextSize.js

```diff
@@ -1,21 +1,38 @@
 import { h } from 'preact'
-import { useI18n, useSoftkey } from 'hooks'
+import { useI18n, useSoftkey, useArticleTextSize } from 'hooks'
+import { articleTextSize } from 'utils'
 
 export const TextSize = ({ close }) => {
   const i18n = useI18n()
 
+  const [textsize, setTextSize] = useArticleTextSize()
+  const { onKeyArrowLeft, onKeyArrowRight } = articleTextSize.getSoftkeyEffect(setTextSize)
+  const sliderValue = ['0', '16.6', '33.2', '49', '66.6', '83.2', '94']
```

Comment:
when looking at this value on the UI, it seems not very equally distribute in the position, and I can see the difficulty here, so let's think about it after merging.

Reply:
Yea I populated `sliderValue` with a mix of math and eye measuring, indeed something we can fine tune later